t/t2801-top-cmd.t

```diff
@@ -10,8 +10,6 @@ runpty="${SHARNESS_TEST_SRCDIR}/scripts/runpty.py"
 waitfile="${SHARNESS_TEST_SRCDIR}/scripts/waitfile.lua"
 testssh="${SHARNESS_TEST_SRCDIR}/scripts/tssh"
 
-export FLUX_URI_RESOLVE_LOCAL=t
```

Comment:
This is fine, but might actually be more tractable long term to just fix the one test test that unsets the variable to run in a sub-shell, since every case of resolving a URI in the test should use `FLUX_URI_RESOLVE_LOCAL=t` (and it might not always be clear that a test will end up resolving a uri...)

Reply:
good idea.  This change was from PR #5032 so I'll push a change in that PR.

Reply:
Oh, oops sorry!